sdk/src/types.ts

```diff
@@ -645,6 +657,7 @@ export type SpotMarketAccount = {
 	nextFillRecordId: BN;
 	spotFeePool: PoolBalance;
 	totalSpotFee: BN;
```

Comment:
is it worth adding the other spot market account assets here and the dlob/helpers.ts?

Reply:
yeah